README.md

```diff
@@ -168,6 +169,10 @@ The payload is expected to be JSON with the following fields:
 - `dry_run` - If set to true, only print out the image digests and tags that would be deleted.
   Useful to test the result is what's expected.
 
+- `concurrency` - Set the number of concurrent deletions. The default is equal to the number of CPUs.
```

Comment:
say that its optional

Reply:
`The default is equal to the number of CPUs.` That's not enough?
All the others are optional too. I would need to change them all

Reply:
I can rephrase it more like the others:
```
If set, this will be the number of deletions that will happen in parallel. The default is equal to the number of CPUs.
```

Reply:
I could add a line:
```
If set, this will be the number of deletions that will happen in parallel. The default is equal to the number of CPUs.
Useful in case the Google rate limiter is being hit.
```